<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Submarine</class>
 <widget class="QMainWindow" name="Submarine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Submarine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>800</y>
      <width>397</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="forward">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(17, 255, 0);</string>
       </property>
       <property name="text">
        <string>Throttle Up</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="left">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 169, 19);</string>
         </property>
         <property name="text">
          <string>Rudder Left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Throttle Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="right">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 169, 19);</string>
         </property>
         <property name="text">
          <string>Rudder Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>800</y>
      <width>95</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="up">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 193, 193);</string>
       </property>
       <property name="text">
        <string>Elevator Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="down">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 193, 193);</string>
       </property>
       <property name="text">
        <string>Elevator Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="infobutton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>800</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 193, 193);</string>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="rpmLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>339</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="currentLayout"/>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>800</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="voltageLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>570</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="speedLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>330</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="pumpLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>50</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="headingLayout"/>
   </widget>
   <zorder>videoWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>infobutton</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>textBrowser</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>horizontalLayoutWidget_4</zorder>
   <zorder>horizontalLayoutWidget_5</zorder>
   <zorder>horizontalLayoutWidget_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
